ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewBackgroundDrawable.java

```diff
@@ -791,7 +791,7 @@ private void updatePath() {
 
     /** Compute mInnerTopLeftCorner */
     mInnerTopLeftCorner.x = mInnerClipTempRectForBorderRadius.left;
-    mInnerTopLeftCorner.y = mInnerClipTempRectForBorderRadius.top;
+    mInnerTopLeftCorner.y = mInnerClipTempRectForBorderRadius.top * 2;
```

Comment:
Can you go into detail about why we multiply by 2?

Reply:
So basically, the reason for applying with 2 is that, the `mInnerClipTempRectForBorderRadius` without multiplying doesn't really curves the border at the end, which leaves the white space around the view. As you can see in the attached image and the issue as well. So after debugging it clicked that if the `mInnerClipTempRectForBorderRadius` can have twice the value we can have a nice curve around borders.